src/assets/i18n/en.json5

```diff
@@ -349,6 +349,10 @@
 
   "admin.access-control.epeople.form.table.collectionOrCommunity": "Collection/Community",
 
+  "admin.access-control.epeople.form.table.remove": "Remove",
+
+  "admin.access-control.epeople.form.table.edit.buttons.removegroup": "Remove member from group \"{{name}}\"",
```

Comment:
These two i18n keys are not used at this time.  But, I suspect it is because there's code missing from your `eperson-form.component.html` file.